routing/localchans/manager.go

```diff
@@ -180,6 +208,87 @@ func (r *Manager) UpdatePolicy(newSchema routing.ChannelPolicy,
 	return failedUpdates, nil
 }
 
+// createEdge recreates an edge and policy from an open channel in-memory.
+func (r *Manager) createEdge(channel *channeldb.OpenChannel) (
+	*models.ChannelEdgeInfo, *models.ChannelEdgePolicy, error) {
+
+	var (
+		nodeKey1Bytes, nodeKey2Bytes,
+		bitcoinKey1Bytes, bitcoinKey2Bytes []byte
+		featureBuf bytes.Buffer
+	)
+
+	localBytes := r.LocalPubkey.SerializeCompressed()
+	remoteBytes := channel.IdentityPub.SerializeCompressed()
+	localBitcoin := channel.LocalChanCfg.MultiSigKey.PubKey.
+		SerializeCompressed()
+	remoteBitcoin := channel.RemoteChanCfg.MultiSigKey.PubKey.
+		SerializeCompressed()
+
+	// local1 is a value indicating whether the local node is node 1
+	// in the channel edge. If true, the local node is node 1, if
+	// false the local node is node 2.
+	local1 := bytes.Compare(localBytes, remoteBytes) < 0
```

Comment:
since there's no `local2` i think we can just name it `local`

Reply:
Done